### PR TITLE
Auto populate user email on support page + UI build

### PR DESCRIPTION
**Pull request by LinusCastelino**

### CODE REVIEW

File: src/main/react-front-end/src/views/Support/SupportComponent.js [hunk 1]
```diff
@@ -95,7 +96,8 @@ export default class SupportComponent extends Component{
               <TextField
                 required
                 label = 'Email Address'
-                name = 'email' 
+                name = 'email'
+                value = { cookies.get('email') }
```
Comment (yinyifu) [inline src/main/react-front-end/src/views/Support/SupportComponent.js]: Can you get it from store.getState().email instead of cookies

Comment (LinusCastelino) [reply src/main/react-front-end/src/views/Support/SupportComponent.js]: Makes sense to get it from store state. Thanks for pointing out. Will do it in the evening.

Comment (LinusCastelino) [reply src/main/react-front-end/src/views/Support/SupportComponent.js]: Apologies for the delay. Change checked in. Please review.

**Comment by LinusCastelino** [comment]
Done..
Not sure why we store the literal in store though. Ideally it should be blank. No checks would be needed then.

**Comment by yinyifu** [comment]
Actually, you are right. email literal serves no purpose. Anyways thanks.

**Review by yinyifu — APPROVED** [review]
approved

**Pull request merged**